services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/MessageManager.java

```diff
@@ -131,11 +127,7 @@ public void sendNewPasswordResetEmail(String passwordResetPrefix, PasswordResetS
 	public void sendDeliveryFailureEmail(String messageId, List<String> errors) throws NotFoundException;
```

Comment:
Marco wants to not send delivery failures for "notification emails" as those "should" be system auto-sent.